src/reducers/index.js

```diff
@@ -257,6 +257,36 @@ const slice = createSlice({
   },
 });
 
+function initializeFixedAndScrollableColumnsCount(state) {
```

Comment:
Can you combine this with `initializeFixedColumnWidthsAndOffsets`? This'll avoid fetching the fixed columns twice.